src/System.Private.CoreLib/src/System/Threading/LowLevelThread.cs

```diff
@@ -18,22 +19,16 @@ namespace System.Threading
 {
     internal static class LowLevelThread
```

Comment:
Is there a point of having this class? RuntimeThread.Windows.cs maybe a better home for whatever is left here.